cognite_toolkit/_cdf_tk/commands/build.py

```diff
@@ -268,11 +274,16 @@ def build_modules(
             last_identifiers: set[tuple[Hashable, Path]] = set()
             for iteration, module_variables in enumerate(module_variable_sets, 1):
                 try:
-                    built_module_resources = self._build_module_resources(module, build_dir, module_variables, verbose)
+                    built_module_resources = self._build_module_resources(
+                        module, build_dir, module_variables, verbose, validation
+                    )
                 except ToolkitError as err:
                     if on_error == "raise":
                         raise
 
+                    if validation == "identifier":
```

Comment:
Will not this lead to a silent skip on_error =="continue". The code below logs what the issue was with building the module resources.